src/config/Config.zig

```diff
@@ -1149,6 +1149,14 @@ keybind: Keybinds = .{},
 ///   * `end` - Insert the new tab at the end of the tab list.
 @"window-new-tab-position": WindowNewTabPosition = .current,
 
+/// Background color for the window titlebar. This only takes effect if
+/// window-theme is set to ghostty.
+@"window-titlebar-background": ?Color = null,
+
+/// Foreground color for the window titlebar. This only takes effect if
+/// window-theme is set to ghostty.
+@"window-titlebar-foreground": ?Color = null,
```

Comment:
Easy change but we should note for each of this that this configuration only applies to GTK currently.

Reply:
I'll go ahead and add that. Was hoping I could snipe someone into adding the feature

Reply:
It's non-trivial for macOS since we have a variety of titlebar styles that all need to be implemented separately I think.